docs/README.md

```diff
@@ -195,10 +195,8 @@ For details see
 Useful tools that can also be used outside CodeChecker.
 
 * [Build Logger (to generate JSON Compilation Database from your builds)](/analyzer/tools/build-logger/README.md)
-* [Plist to HTML converter (to generate HTML files from the given plist files)](/docs/tools/plist_to_html.md)
```

Comment:
The documentation is moved to here: https://github.com/Ericsson/codechecker/blob/master/docs/tools/report-converter.md#plist-to-html-tool